apps/api/v1/pages/api/event-types/_utils/checkTeamEventEditPermission.ts

```diff
@@ -10,21 +10,17 @@ export default async function checkTeamEventEditPermission(
   req: NextApiRequest,
   body: Pick<z.infer<typeof schemaEventTypeCreateBodyParams>, "teamId" | "userId">
 ) {
-  const { isSystemWideAdmin } = req;
-  let userId = req.userId;
-  if (isSystemWideAdmin && body.userId) {
-    userId = body.userId;
-  }
```

Comment:
We really don't need this check here again primarily because isSystemWideAdmin means it's all clear for the access, and we do that everywhere this is called. So it's actually never really succeeds here lol (because it only reaches here when isSystemWideAdmin is false) 🫠